src/words/ast/INodeCreateClass.java

```diff
@@ -10,7 +15,40 @@ public INodeCreateClass(Object... children) {
 
 	@Override
 	public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException {
-		// TODO
-		throw new AssertionError("Not yet implemented");
+		ASTValue className  = this.children.get(0).eval(environment);
+		ASTValue parentClassName = this.children.get(1).eval(environment);
+		
+		WordsClass wordsClass = environment.createClass(className.stringValue, parentClassName.stringValue);
+		
+		INodeClassStatementList statementList = (INodeClassStatementList) this.children.get(2);
+		if (statementList != null) {
+			for (Object child : statementList.children) {
+				if (child instanceof INodeDefineProperty) {
+					ASTValue propertyName = ((INodeDefineProperty) child).id.eval(environment);
+					
+					if (((INodeDefineProperty) child).literal == null) {
+						wordsClass.setProperty(propertyName.stringValue, new WordsProperty(WordsProperty.PropertyType.NOTHING));
+					} else {
+						ASTValue propertyValue = ((INodeDefineProperty) child).literal.eval(environment);
```

Comment:
I've implemented a method that simplifies this.  It happens to be on the queue-prop-assign-branch in #87.  I think it would be good if you merged that branch into this one and used that refactor.  (Sorry that it's not on a separate branch by itself, which would have been better practice on my part.)